CONTRIBUTING.md

```diff
@@ -3,15 +3,46 @@ welcome contributions from everyone.  Below are various bits of information to
 help you get started.  If you require additional help, please reach out to us on
 our [zulip channel](https://near.zulipchat.com/).
 
-## Getting started
+## Quick Start
 
-- For non-technical contributions, such as e.g. content or events, see [this
-document](https://docs.nearprotocol.com/docs/contribution/contribution-overview).
+nearcore is a fairly standard Rust project, so building as easy as
 
-- For an overview of the NEAR core architecture, see [this
-playlist](https://www.youtube.com/playlist?list=PL9tzQn_TEuFV4qlts0tVgndnytFs4QSYo).
+```console
+$ cargo build
+```
+
+Building nearcore requires a fairly recent Rust compiler (get it
```

Comment:
It would, but there's a high chance that you already have the required stuff installed. 

Can't really articulate why, but the flow of "try building, here's how you debug this" over "here's the list of things you need to do beforehand for the build to succeed" is intentional.

If I try to rationalize that, it's because we don't *really* have universal instructions which are guaranteed to work after copy-paste (aka, not everyone is using Ubuntu). So, we are fundamentally in a "some assembly required" situation, and for that I think the right workflow is to do a trial build, and then install what's missing.

Reply:
Going to merge as is, but no strong opinion! Would merge any clarifications!

Reply:
Yeah, please go ahead.  I don't have strong feelings one way or other.  In practice the errors returned will also be pretty obvious.